recipes/parsmurf/meta.yaml

```diff
@@ -0,0 +1,66 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "parSMURF" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/AnacletoLAB/{{ name }}/releases/download/v{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 426013313b22d8fdd0db907e920e869347fb53e9e25cd59f07adc4b2eaf51ad7
+
+build:
+  skip: true  # [win]
+  number: 0
+
+requirements:
+  build:
+    # If your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+    # Compilers are named 'c', 'cxx' and 'fortran'.
+    - {{ compiler('cxx') }}
+    - cmake >=2.8
+    - openmpi
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+  host:
+    - numpy ==1.13.0
+    - scipy ==1.2.1
+    - weave ==0.17.0
+    - six ==1.12.0
+    - protobuf ==3.7.1
+    - openmpi
+  run:
+    - numpy ==1.13.0
+    - scipy ==1.2.1
+    - weave ==0.17.0
+    - six ==1.12.0
+    - protobuf ==3.7.1
```

Comment:
Do all of these need to be pinned so tightly?

Reply:
I try to make them more flexible